app/api/event_copy.py

```diff
@@ -45,13 +46,22 @@ def create_event_copy(identifier):
     discount_codes = DiscountCode.query.filter_by(event_id=event.id).all()
     speaker_calls = SpeakersCall.query.filter_by(event_id=event.id).all()
     user_event_roles = UsersEventsRoles.query.filter_by(event_id=event.id).all()
+    taxes = Tax.query.filter_by(event_id=event.id).all()
```

Comment:
All these queries should have `deleted_at=None`

Reply:
@iamareebjamal understood, will do this.